popUp.js

```diff
@@ -0,0 +1,412 @@
+              <li class="weight-desktop">Back End Dev</li>
+              <li>
+                <i class="fa-solid fa-circle fa-2xs project-role-counter"></i>
+              </li>
+              <li class="weight-desktop">2015</li>
+            </ul>
+            <p class="project-description-paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="project-language-list">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="button-container">
+              <button type="button" id="mobile3" class="see-project-button get-button mobile-display">See project</button>
+              <button type="button" id="desktop1" class="button-desktop big-size get-button">See project</button>
+            </div>
+          </div>
+        </article>
+      
+        <article id="card-work-4" class="mobile-card">
+          <img
+            class="card-snapshot"
+            src="Images_Background/Snapshoot Portfolio 4_Art Printing.svg"
+            alt="Project 4 Snapshot"
+          />
+          <div class="project-description-block">
+            <h2 class="card-title mobile-display">Tonic</h2>
+            <h2 class="card-title big-size">Multi-Post Stories</h2>
+            <ul class="project-role-detail">
+              <li class="project-role-client mobile-display">CANOPY</li>
+              <li class="project-role-client big-size">FACEBOOK</li>
+              <li>
+                <i class="fa-solid fa-circle fa-2xs project-role-counter"></i>
+              </li>
+              <li class="mobile-display">Back End Dev</li>
+              <li class="big-size weight-desktop">Full Stack Dev</li>
+              <li>
+                <i class="fa-solid fa-circle fa-2xs project-role-counter"></i>
+              </li>
+              <li class="weight-desktop">2015</li>
+            </ul>
+            <p class="project-description-paragraph mobile-display">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <p class="project-description-paragraph big-size">
+              Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+            </p>
+            <ul class="project-language-list">
+              <li>HTML</li>
+              <li class="big-size">Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="button-container">
+              <button type="button" id="mobile4" class="see-project-button get-button mobile-display">See project</button>
+              <button type="button" id="desktop2" class="button-desktop big-size get-button">See project</button>
+            </div>
+          </div>
+        </article>`;
```

Comment:
- [ ] As per the requirements the data for this section (project section) should be created from the projects array, kindly make sure to fix this so it gets the data dynamically from the array, this will make it easier to add any new project to your section